<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormInputOutputChord</class>
 <widget class="QWidget" name="FormInputOutputChord">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>312</width>
    <height>124</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="groupBoxInput">
       <property name="title">
        <string>Origin</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QLabel" name="labelChordInput">
              <property name="text">
               <string>Chord</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="labelMinorInput">
              <property name="text">
               <string>Minor</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QComboBox" name="comboBoxInputChordLanguage"/>
            </item>
            <item>
             <widget class="QComboBox" name="comboBoxMinorInput"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBOutput">
       <property name="title">
        <string>Destination</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QLabel" name="labelChordOutput">
              <property name="text">
               <string>Chord</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="labelMinorOutput">
              <property name="text">
               <string>Minor</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <widget class="QComboBox" name="comboBoxOutputChordLanguage"/>
            </item>
            <item>
             <widget class="QComboBox" name="comboBoxMinorOutput"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
